src/types/external.ts

```diff
@@ -31,6 +31,12 @@ export interface InlineConfig {
    * @default "${config.srcDir}/entrypoints"
    */
   entrypointsDir?: string;
+  /**
+   * Specify allow entrypoints, default allow all entrypoints
+   *
+   * @default true
+   */
+  allowEntrypoints?: true | string[];
```

Comment:
I'd prefer just `string[]` here. True is the same as undefined, I don't want more than one type to have the same meaning.

```suggestion
  allowEntrypoints?: string[];
```